doc/md_src_files/059_coordination-publish.md

```diff
@@ -0,0 +1,51 @@
+## Publication Decision
+
+A coordinator often has to decide when or whether to publish information about a vulnerability.
+A supplier also makes a decision about publicity -- releasing information about a remediation or mitigation could be viewed as a kind of publication decision.
+While the context of publication is different for coordinators, a supplier may find some use in a publication decision if they need to decide whether to publish before a mitigation or remediation is available.
+However, that is not the intended use case; this section describes how a coordinator might decide to publish information about a vulnerability.
+
+The publication decision is always a decision at a point in time.
+As discussed in [Guidance on Communicating Results](#guidance-on-communicating-results), any SSVC decision may change over time as the inputs to that decision change.
+Therefore, a decision to not publish is a decision not to publish at the time the decision was made.
+It is not a decision never to publish in the future.
+One benefit of encoding the decision process in SSVC is the analysts can all agree on what new information would change the decision and prioritize maintaining awarenss of just those decision points.  
+
+This section is based on CERT/CC policy choices.
+Two points where this clearly influences the publication decision are embargo periods and scope.
+As a matter of policy, CERT/CC will not choose to publish information about a vulnerability while the existence of that vulnerability is under embargo.
+The default embargo period is [45 days](https://vuls.cert.org/confluence/display/Wiki/Vulnerability+Disclosure+Policy).
+There are various ways an embargo period might be broken, including active exploitation of the vulnerability in the wild, but the decision described in this section assumes the embargo period is over, one way or another.
```

Comment:
I'm a little put off by the phrasing around embargoes here. Mainly my concern is that breaking the embargo is a choice made to behave a certain way _after_ one or more indicator events have occurred. (timer expired, exploitation observed, public discussion, etc.) The indicator events themselves do not break the embargo automatically.

Suggested change:
As a matter of policy, CERT/CC will support an embargo information regarding the existence of a vulnerability
by through its choice not to publish information about the vulnerability while the a number of conditions hold:

1. A negotiated embargo timer has not expired. The default embargo period is [45 days](https://vuls.cert.org/confluence/display/Wiki/Vulnerability+Disclosure+Policy).
2. Other exceptions have not been met, including active exploitation of the vulnerability in the wild or other public discussion of the vulnerability details.

Regardless, the decision described in this section assumes the embargo period is over, one way or another.

Reply:
resolved by aaf85f30db5c80b094e8dbd599a4b2ee614adff0